BitcoinRegressionProject.ipynb

```diff
@@ -61,172 +61,568 @@
        " 2021-12-30         0.8     NaN   0.8   0.8  0.8)"
       ]
      },
-     "execution_count": 32,
+     "execution_count": 354,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
     "import pandas as pd \n",
-    "import os\n",
     "import numpy as np\n",
     "\n",
+    "import os\n",
+    "\n",
     "os.chdir(r'C:\\Users\\nicho\\OneDrive\\Desktop\\Bitcoin Regresssion Project')\n",
-    "BTC = pd.read_csv(r'BTC-USD.csv' , parse_dates = ['Date'], index_col=['Date'])\n",
-    "SNP = pd.read_csv(r'Standard&PoorsHistorical.csv', parse_dates = ['Date'], index_col=['Date'])\n",
-    "Margin = pd.read_csv(r'margin-statistics.csv')\n",
-    "CPI = pd.read_csv(r'CPIHistoricalData.csv', parse_dates = ['Date'], index_col= ['Date'])\n",
-    "BTC.head(), Margin.head(), SNP.head(), CPI.head()"
+    "\n",
+    "btc = pd.read_csv('BTC-USD.csv', parse_dates=['Date'], index_col=['Date'])\n",
+    "snp = pd.read_csv('Standard&PoorsHistorical.csv', parse_dates=['Date'], index_col=['Date'])\n",
+    "margin = pd.read_csv('margin-statistics.csv')\n",
+    "cpi = pd.read_csv('CPIHistoricalData.csv', parse_dates=['Date'], index_col=['Date'])\n",
+    "\n",
+    "btc.head(), snp.head(), margin.head(), cpi.head()"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 33,
+   "execution_count": 355,
    "metadata": {},
    "outputs": [],
    "source": [
     "#Dropping irrelevant features\n",
     "\n",
-    "Drop = ['Symbol', 'Open', 'High', 'Low', 'Volume BTC']\n",
-    "BTCDrop = BTC.drop(Drop, axis = 1)\n",
+    "drop_btc_cols = ['Symbol', 'Open', 'High', 'Low', 'Volume BTC']\n",
+    "btc.drop(drop_btc_cols, axis=1, inplace=True)\n",
```

Comment:
Instead of naming things like `Drop1`, `Drop2`, etc, it's better to give context of what those variables are used to do. So in this case `Drop` -> `drop_btc_cols` makes more sense. Also, this helps 1000% when you comeback and read this code in 3 months haha